channels-rke2.yaml

```
@@ -2168,7 +2168,7 @@ releases:
     featureVersions: *featureVersions-v1
   - version: v1.28.14+rke2r1
     minChannelServerVersion: v2.8.3-alpha1
-    maxChannelServerVersion: v2.10.99
+    maxChannelServerVersion: 2.9.99
```

Should have a `v` prefix I believe?
```suggestion
    maxChannelServerVersion: v2.9.99
```